extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1231,15 +1231,9 @@ static public function getLocalEnvURL( $url, $forcedEnv = null ) {
 
 		// strip env-specific pre- and suffixes for staging environment
 		$server = preg_replace( '/^(preview|verify|sandbox-[a-z0-9]+)\./', '', $server );
-		$devboxRegex = '/\.([^\.]+)\.wikia-dev\.com$/';
-
-		if ( preg_match( $devboxRegex, $server, $groups ) === 1 ) {
-			$devbox = $groups[ 1 ];
-		} else {
-			$devbox = '';
+		if ( !empty( $wgDevDomain ) ) {
+			$server = str_replace( ".{$wgDevDomain}", '', $server );
 		}
-
-		$server = str_replace( '.' . $devbox . '.wikia-dev.com', '', $server );
 		$server = str_replace( static::WIKIA_TOP_DOMAIN, '', $server );
 		$server = str_replace( '.' . $wgWikiaBaseDomain, '', $server ); // PLATFORM-2400: make WF redirects work on staging
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)